pyaerocom/io/read_eea_aqerep.py

```diff
@@ -555,14 +605,8 @@ def read(self, vars_to_retrieve=None,
     # Test that the reading routine works
     from pyaerocom.io.read_eea_aqerep import ReadEEAAQEREP
     import logging
-    r = ReadEEAAQEREP()
-    r.logger.setLevel(logging.INFO)
-    var_names_to_test = ['concpm25', 'vmro3','concpm10', ]
-    station_id = 'AT2KA71'
-    for var_name in var_names_to_test:
-        data = r.read(vars_to_retrieve = [var_name])
-        print('data read')
-        print(data[station_id])
-        print(data[station_id][var_name])
+    ddir = '/home/jonasg/MyPyaerocom/data/obsdata/EEA_AQeRep.NRT/download'
```

Comment:
we should find a way for testing that does not replace each others testing routines...
This is not ideal since I will add mine with the newest commit again...

Reply:
Sorry that I removed your code. I think testing should not happen under `__main__`, which is for playing around and changes all the time also in all other pyaerocom modules. If we want to test some code, we should do this in a separate python file. Best would be to create a "representative" small subset (best <1MB) of the data and add it to testdata-minimal and add a test module `pyaerocom/io/test/test_read_eea_aqerep.py` that runs tests based on that subset.